lib/Agrammon/Documentation.pm6

```diff
@@ -27,13 +28,12 @@ sub create-latex-source( Str $model-name, Agrammon::Model $model, $sort, :%techn
 
     my $last-section = '';
     for @sections -> %section {
+        next unless @(%section<inputs>).elems or @(%section<outputs>).elems;
```

Comment:
Guess I missed this before, but it's probably a little over the top. I assume `%section<inputs>` already is an array, so the `@` is just to strip away the scalar container around it, which one would need to do when iterating it with `for`. But methods always see through the container, so `%section<inputs>.elems` works just fine anyway. But there's more: putting something in boolean context works by calling its `Bool` method, which *also* sees through the container, so `next unless %section<inputs> or %section<outputs>;` will do, at which point we might shorten it to `next unless any %sections<inputs outputs>`.